src/Datadog.Trace.ClrProfiler.Native/environment_variables_util.h

```diff
@@ -0,0 +1,79 @@
+#ifndef DD_CLR_PROFILER_ENVIRONMENT_VARIABLES_UTIL_H_
```

Comment:
@gleocadie Would you mind checking this file? I don't expect any error but I'm not familiar with macros

Reply:
LGTM